src/vfs.js

```diff
@@ -73,6 +73,16 @@ const handleDirectoryList = (path, options) => result =>
       showHiddenFiles: options.showHiddenFiles !== false,
       filter: options.filter
     }));
+/**
```

Comment:
Could you add a new line between the previous code and this function definition?